src/test/translation_tests.cpp

```diff
@@ -9,13 +9,21 @@
 
 BOOST_AUTO_TEST_SUITE(translation_tests)
 
+static TranslateFn translate{[](const char * str) {  return strprintf("t(%s)", str);  }};
+
 BOOST_AUTO_TEST_CASE(translation_namedparams)
 {
+    constexpr util::TranslatedLiteral format{"original [%s]", &translate};
```

Comment:
In commit "refactor: Check translatable format strings at compile-time" (fa948257f332772f058cabffe0e313a0518da724)

Could consider making the test a little more realistic with:

```diff
--- a/src/test/translation_tests.cpp
+++ b/src/test/translation_tests.cpp
@@ -11,17 +11,22 @@ BOOST_AUTO_TEST_SUITE(translation_tests)
 
 static TranslateFn translate{[](const char * str) {  return strprintf("t(%s)", str);  }};
 
+// Custom translation function _t(), similar to _() but internal to this test.
+static inline consteval auto _t(util::TranslatedLiteral str)
+{
+    str.translate_fn = &translate;
+    return str;
+}
+
 BOOST_AUTO_TEST_CASE(translation_namedparams)
 {
-    constexpr util::TranslatedLiteral format{"original [%s]", &translate};
-
     bilingual_str arg{"original", "translated"};
-    bilingual_str result{strprintf(format, arg)};
+    bilingual_str result{strprintf(_t("original [%s]"), arg)};
     BOOST_CHECK_EQUAL(result.original, "original [original]");
     BOOST_CHECK_EQUAL(result.translated, "t(original [translated])");
 
     util::TranslatedLiteral arg2{"original", &translate};
-    bilingual_str result2{strprintf(format, arg2)};
+    bilingual_str result2{strprintf(_t("original [%s]"), arg2)};
     BOOST_CHECK_EQUAL(result2.original, "original [original]");
     BOOST_CHECK_EQUAL(result2.translated, "t(original [t(original)])");
 }
```